nzshm_common/location/location.py

```diff
@@ -63,7 +63,7 @@
 }
 
 
-def _lat_lon(_id) -> Optional[LatLon]:
+def lat_lon_by_id(_id) -> Optional[LatLon]:
```

Comment:
@chrisdicaprio actually I'm confused why we'd elevate this _method anyway? -  isn't the preferred way the location_by_id() function??

Reply:
I think we're both wrong. I should be using `get_locations`. The function name isn't obvious for the application, but it will return a list of `CodedLocations` given a list of location ids. This is used in `runzi` `toshi-hazard-post`, and the TS1170 lib.